CHANGELOG.md

```diff
@@ -41,7 +41,8 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
   - For questions about this change, please contact the Red Hat support team at support@redhat.com.
 - ROX-10018: The policy `OpenShift: Kubeadmin Secret Accessed` will no longer trigger if the request was from the default OpenShift `oauth-apiserver-sa` service account, because this is an expected access pattern for the OpenShift apiserver.
 - Violation tags and process tags are deprecated, and will be removed in version 3.72.0.
-
+- RBAC calculation is no longer included in Risk by default, since it was not performant. Users who want it included can set
+  the "INCLUDE_RBAC_IN_RISK" to "true" in the Central deployment spec.
```

Comment:
I think the default should be true, and set to false if not required? I believe Kirsten had the same feedback. In her exact words:
"Also, it should not be disabled by default so as not to take aways current functionality"